src/interfaces/vkd3d-proton_interfaces.h

```diff
@@ -130,7 +130,6 @@ ID3D12DXVKInteropDevice : public IUnknown {
 };
 MIDL_INTERFACE("902d8115-59eb-4406-9518-fe00f991ee65")
 ID3D12DXVKInteropDevice1 : public ID3D12DXVKInteropDevice {
-
```

Comment:
Unrelated but if you're touching this, could you please clang-format interface headers, even if that requires temporarly getting rid of `MIDL_INTERFACE`s to unconfuse it? That `TYPE * name` is starting to hurt my eyes.

Reply:
not part of this pr, but yeah agree ;)